jupyter_book/directive/toc.py

```diff
@@ -22,29 +29,154 @@ def run(self):
         return [TableOfContentsNode()]
 
 
-class SwapTableOfContents(SphinxTransform):
-    default_priority = 700
+def process_toc_dict(
```

Comment:
Is this for use with `tableofcontents` or something? Does the apply to non-latex usecases or is it just for latex?

Reply:
It is just for latex use case at the moment. The `SwapTableofContents` post-transform handles latex and non-latex differently. This function is used only in latex case. https://github.com/executablebooks/jupyter-book/pull/1167/files#diff-d6d834bc60c86a22e11aed333d22d8ac5f9eb097f8da28d3c299da8e2806adf5R137